model/src/cfd.rs

```diff
@@ -1474,7 +1552,7 @@ impl Dlc {
     /// Create a close transaction based on the current contract and a settlement proposals
     pub fn close_transaction(
         &self,
-        proposal: &crate::model::cfd::SettlementProposal,
+        proposal: &crate::cfd::SettlementProposal,
```

Comment:
As a folllow-up, I think we should flatten this crate and then extract more meaningful modules. This `cfd` submodule is annoying me 🤣